app/src/main/res/layout/fragment_news_list.xml

```diff
@@ -0,0 +1,14 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".NewsListFragment">
```

Comment:
What's the use of `tools:context` here? I tried to delete it and the layout still worked.